### PR TITLE
Dev/pg v13

### PR DESCRIPTION
**Pull request by danperezsan**
Add full backup schedule process using cron job scheduler

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ To run backups and WAL archiving to GCS (Google Cloud Storage) set the following
       - BACKUPS=true                                            # switch to implement backups; defaults to false
       - STORAGE_BUCKET=gs://postgresql13/wal-g                  # to specify the GCS bucket
       - GCP_CREDENTIALS=/run/secrets/gcp_credentials            # to specify the docker secret with the service account key that has access to the GCS bucket
+      - CRON_SCHEDULE:* * * * *                                 # to specify the cron schedule expression at which backups will run (if not set only the first initial base backup will be ran) \
```
Comment (gbmeuk) [inline README.md:48]: Please call the FULL_BACKUP_SCHEDULE

File: scripts/entrypoint.sh [hunk 2]
```diff
@@ -35,6 +36,16 @@ if [[ ${RESTORE_BACKUP^^} == TRUE && -z ${BACKUP_NAME} ]]; then
   exit 1
 fi
 
+if [[ ${BACKUPS^^} == TRUE ]] && [[ ! -z ${CRON_SCHEDULE}  ]] && [[ $(id -u) == 0 ]]; then
+  echo "Starting cron job scheduler" && crond
+  echo "Database backups will be scheduled to run at ${CRON_SCHEDULE}. Check https://crontab.guru/ for schedule expression details"
+fi
+
+function cron_schedule() {
+    CRON_CONFIGURATION="${CRON_SCHEDULE} /usr/local/scripts/base_backup.sh >> /var/log/pg-backups.log"
```
Comment (gbmeuk) [inline scripts/entrypoint.sh:45]: Does this not need to be going to standard out?

**Review by gbmeuk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (danperezsan) [reply README.md]: Done

File: scripts/entrypoint.sh (hunk 2 above)
Comment (danperezsan) [reply scripts/entrypoint.sh]: Done

**Comment by danperezsan** [comment]
Added cronitor installation and configuration

File: scripts/entrypoint.sh [hunk 3]
```diff
@@ -139,6 +145,19 @@ function restore_backup() {
     docker_temp_server_stop
 }
 
+if [[ ${BACKUPS^^} == TRUE ]] && [[ ! -z ${FULL_BACKUP_SCHEDULE}  ]] && [[ $(id -u) == 0 ]]; then
+  echo "Starting cron job scheduler" && crond
+  echo "Database backups will be scheduled to run at ${FULL_BACKUP_SCHEDULE}. Check https://crontab.guru/ for schedule expression details"
+  backup_cron_schedule
+  if [[ ! -z ${CRONITOR_KEY} ]]; then
+    echo "Installing and configuring cronitor. Check https://cronitor.io/cron-job-monitoring to see jobs monitoring"
+    curl -sOL https://cronitor.io/dl/linux_amd64.tar.gz
```
Comment (gbmeuk) [inline scripts/entrypoint.sh:154]: Let's not do this every time a container starts. Put it into the Dockerfile

**Review by gbmeuk — CHANGES_REQUESTED** [review]

**Comment by danperezsan** [comment]
Redirected cronitor configure stdout to /dev/null

**Pull request merged**